tests/features/project.feature

```diff
@@ -0,0 +1,25 @@
+@api @project
```

Comment:
What does project tag do? Please remove if there is no associated logic to that.